app/models/manageiq/providers/openstack/infra_manager/orchestration_stack.rb

```diff
@@ -1,12 +1,12 @@
 class ManageIQ::Providers::Openstack::InfraManager::OrchestrationStack < ::OrchestrationStack
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::InfraManager"
 
-  def raw_update_stack(template, parameters)
+  def raw_update_stack(template, parameters, clear_parameters = [])
```

Comment:
Can you change the argument list to `raw_update_stack(template, options)`? Please follow the pattern in https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/orchestration_stack.rb#L15

This requires a search for all calls to this method and update accordingly.